lightning/src/ln/channel.rs

```diff
@@ -9811,6 +9821,7 @@ impl<SP: Deref> Writeable for FundedChannel<SP> where SP::Target: SignerProvider
 			(49, self.context.local_initiated_shutdown, option), // Added in 0.0.122
 			(51, is_manual_broadcast, option), // Added in 0.0.124
 			(53, funding_tx_broadcast_safe_event_emitted, option), // Added in 0.0.124
+			(55, self.is_v2_established, option), // Added in 0.2
```

Comment:
Seems we can use `default_value` with `false` and not use an `Option`?

Reply:
You are right in that we don't actually need this and it can be inferred from the ChannelId.